source/ota.c

```diff
@@ -2311,21 +2312,20 @@ static IngestResult_t ingestDataBlockCleanup( OtaFileContext_t * pFileContext,
         if( pFileContext->pFile != NULL )
         {
             *pCloseResult = otaAgent.pOtaInterface->pal.closeFile( pFileContext );
+            otaPalMainErr = OTA_PAL_MAIN_ERR( *pCloseResult );
+            otaPalSubErr = OTA_PAL_SUB_ERR( *pCloseResult );
```

Comment:
Can remove this since it is unused, or log it.

Reply:
Which one? `otaPalMainErr` is in comparison, otaPalSubErr in in log on line 2326.

Reply:
Ah yes, missed otaPalSubErr in 2326.